### DISCUSSION TITLE
🚀 Manifest Update: Our Latest Features Just Dropped!

### DISCUSSION THREAD

**Discussion by SebConejo** [mnfst/llm-gateway#197] [Announcements]
![v-spectral](https://github.com/user-attachments/assets/f3d4cd97-0cbb-4d73-bb30-d7e6c858515c)

Hey Manifest developers! We’re back with some major improvements that are set to make building with Manifest even more powerful and, yes, more fun. 🎉 This one’s packed with new capabilities, including a much-anticipated validation feature and many-to-many relationships!

### 1️⃣ Built-in Validation:
Want to keep a “Name” field filled, make sure emails have an “@”, or enforce positive numbers? You can now set up these validators in no time! [Dive into it here](https://manifest.build/docs/validation)

### 2️⃣ Many-to-Many Relationships:
You asked for it. We built it. The many-to-many relationships are here!. So you can link multiple `Products` to multiple “Categories” for example. [Documentation here](https://manifest.build/docs/relations)

### 3️⃣ File & Image Uploads:
Add [file](https://manifest.build/docs/properties#file) and [image](https://manifest.build/docs/properties#image) uploads to your apps and watch the possibilities with Manifest multiply! 

Wanna see all the details? 👉 [Check the full release notes](https://github.com/mnfst/manifest/releases/tag/4.0.1).

Let us know what you think or what else you'd like to see! Keep those suggestions coming, and let’s build something awesome together. 🔥

Excited to see what you'll build next with Manifest! 🚀 Drop your projects in our [GitHub show&tell section](https://github.com/mnfst/manifest/discussions/categories/show-tell), hop on [Discord](https://discord.gg/FepAked3W7) for any questions, and feel free to report bugs via [issues](https://github.com/mnfst/manifest/issues). We’re here to help and cheer you on!

Happy coding! 🙌

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
